src/main/java/racingcargame/model/car/Car.java

```diff
@@ -0,0 +1,36 @@
+package racingcargame.model.car;
+
+import racingcargame.dto.CarDto;
+
+public class Car implements Comparable<Car> {
+    private final Name name;
+    private final Position position;
+
+    public Car(final String name) {
+        this.name = new Name(name);
+        this.position = new Position();
+    }
+
+    public void moveCar(final int moveValue) {
+        position.move(moveValue);
+    }
+
+    public boolean hasSamePosition(final Car winnerCar) {
+        return position.compareTo(winnerCar.position) == 0;
```

Comment:
보통 compareTo()를 정의한다는건 직접 해당 메서드를 호출하기 보다는 다음과 같은 방식으로 비교하기 위해 정의합니다.
```suggestion
        return position.equals(winnerCar.position);
```

하지만 지금은 이렇게 하면 테스트가 깨져요. 테스트가 통과할 수 있게 한번 고민해보세요 :)

Reply:
닉이 위에서 주셨던 vo 객체의 equals && hashcode 구현에서 힌트를 얻었습니다!ㅎㅎ
carPosition 클래스에 오버라이딩한 equals를 통해 비교하도록 코드를 수정하고 테스트 코드 또한 수정햇습니다!